ES/ESwq001.xml

```diff
@@ -266,28 +323,29 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                   </decoDesc>
                   <additions>
                      <list>
-                        <item xml:id="a1">
-                           <locus target="#185rc">ff. 185rc-187</locus>
-                           <desc>rb: salām-hymns for Mary.</desc>
-                           <quote xml:lang="gez">
-                              <locus target="#185rc">185rc</locus> (below the explicit of the text):
-                              ሰላም፡ ለኪ፡ ማርያም፡  ዘመአዛ፡ አፉኪ፡ እንተ፡ ትጼንዊ፡ በገዳም፡ ወሀቅል፡ …</quote>
-                           <quote xml:lang="en"/> The note, with rubrications, seems to have been
+                        <item xml:id="a1" corresp="ESbqm005#a1">
+                           <locus from="185rc" to="187rb">ff. 185rc-187rb</locus>
+                           <desc type="GuestText">salām-hymns for Mary.</desc>
```

Comment:
LIT2807RepCh91

Reply:
thanks i believe to have carried out all corrections